packages/react-app/.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Can we also include prettier in #38?

Reply:
This is already in the right place, when `yarn lint` is ran it will automatically include `prettier` as well.

Reply:
This is already in the right place, when `yarn lint` is ran it will automatically include `prettier` as well.